spec/services/attribute_asserter_spec.rb

```diff
@@ -61,25 +63,19 @@
         end
 
         context 'authn request does not specify bundle' do
-          it 'contains DEFAULT_BUNDLE' do
-            expect(user.asserted_attributes.keys).to match_array(
-              AttributeAsserter::DEFAULT_BUNDLE + [:uuid]
-            )
+          it 'only returns uuid' do
```

Comment:
This seems like a big business rule change.

For a LOA3 request, previously, no bundle specified in either config or request resulted in the entire LOA3 bundle being returned by default. Now it is just the UUID.

That should be true for LOA1. Did we decide it is also true for LOA3?